app/editor/src/components/form/wysiwyg/styled/Wysiwyg.tsx

```diff
@@ -2,6 +2,7 @@ import styled from 'styled-components';
 
 interface IWysiwygProps {
```

Comment:
Don't duplicate this interface.  Just use the one the component uses.  I should have caught this before.

Reply:
This interface here needs to be deleted.  Use the one we always include in the component.

Reply:
Updated, thanks